packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -219,6 +268,23 @@ const Select = <
     [onChangeProp],
   );
 
+  const removeSelection = useCallback(
+    (itemToRemove: string) => {
+      if (Array.isArray(internalSelectedValues)) {
+        const newValue = internalSelectedValues.filter(
+          (item: string) => item !== itemToRemove,
+        );
+
+        const syntheticEvent = {
+          target: { value: newValue },
+        } as SelectChangeEvent<Value>;
+
+        onChange(syntheticEvent, null);
+      }
+    },
```

Comment:
Suggestion: exit fast to improve readability:
```
(itemToRemove: string) => {
  if (!Array.isArray(internalSelectedValues)) {
    return;
  }
  const newValue = internalSelectedValues.filter(
    (item: string) => item !== itemToRemove,
  );

  const syntheticEvent = {
    target: { value: newValue },
  } as SelectChangeEvent<Value>;

  onChange(syntheticEvent, null);
}
    },
```

Reply:
This was how I originally did it, but @KevinGhadyani-Okta recommended switching it to this way

Reply:
OK, I'll follow up with him. I'd like to understand why he feels this way about it.